admin/scss/bootstrap/_variables.scss

```diff
@@ -403,23 +403,23 @@ $navbar-padding-vertical:           ($spacer / 2);
 
 // Navs
 
-$nav-link-padding:                          12px 8px;
-$nav-link-hover-bg:                         red; //$gray-lighter;
-
-$nav-disabled-link-color:                   $text-muted;
-$nav-disabled-link-hover-color:             $text-muted;
-
-$nav-tabs-border-color:                     #d9dee2;
-
-$nav-tabs-link-border-width:                0;
-$nav-tabs-link-hover-border-color:          green; //$gray-lighter;
-
-$nav-tabs-active-link-hover-bg:             $body-bg;
-$nav-tabs-active-link-hover-color:          $body-color;
-$nav-tabs-active-link-hover-border-color:   #ddd;
-
-$nav-tabs-justified-link-border-color:            #ddd;
-$nav-tabs-justified-active-link-border-color:     $body-bg;
+// $nav-link-padding:                          .5em 1em;
```

Comment:
We'll need some comments at the top of this file in order to describe what these commented out lines mean. I assume you mean that these are the values as defined in _bootstrap.scss, in the node_modules, correct?

The only problem I want to solve is commented out code with no apparent value. If there is a reason, I just need to make sure others who view this file understand the same benefit that you do.